.travis.yml

```diff
@@ -131,7 +131,7 @@ install:
   - python -m pip install -U pip
   - python -m pip --version
   - python -m easy_install -U setuptools
-  - python -m pip install numpy scipy networkx pytest python-igraph jax jaxlib
+  - python -m pip install numpy scipy networkx pytest python-igraph jax jaxlib torch torchvision
```

Comment:
Since it seems that the tests fails only on non-conda environments on mac, for some relatively old python version, I would assume that this is a torch-related issue rather than our problem. I would suggest to skip tests for torch on this platfom and keep it only on conda when on Mac. It should be enough to add an environment variable TEST_TORCH=0 or something in the travis.yml and check it at install time